corehq/apps/fixtures/views.py

```diff
@@ -226,6 +233,30 @@ def _update_items(fields_patches, domain, data_type_id, transaction):
                 )
         setattr(item, "fields", updated_fields)
         transaction.save(item)
+
+    fields_json = "fields"
+    for field_name, patch in fields_patches.items():
+        if "update" in patch:
+            new_field_name = patch["update"]
+            fields_json = JsonSet(
+                JsonDelete(fields_json, field_name),
+                [new_field_name],
+                JsonGet("fields", field_name),
+            )
+        elif "remove" in patch:
+            fields_json = JsonDelete(fields_json, field_name)
+    for field_name, patch in fields_patches.items():
+        if "is_new" in patch:
+            fields_json = JsonSet(fields_json, [field_name], [])
```

Comment:
phew! took me a bit to get my head around this whole nested json editing bit. Great that we can do a bulk update!